### PR TITLE
Higher header

### LINKED ISSUE #102: Make header higher on mobile devices
Currently the header isn't high enough so that on most pages the content overflows the blue background. We should just make it higher for all pages.

img_0262

### PR DESCRIPTION
**Pull request by trabaldini**
closes #102

### CODE REVIEW

File: css/app.scss [hunk 1]
```diff
@@ -185,7 +185,7 @@ ul {
 
 .primary-banner {
   padding: 20px 0;
-  height: 180px;
+  height: 250px;
```
Comment (marcoow) [inline css/app.scss]: This makes the banner higher for all devices while the height should only be increased on mobile devices. See below - there's a section for devices with a specific max width which targets mobile devices with small screens.

**Review by marcoow — CHANGES_REQUESTED** [review]
We need to limit this change to devices with small screens.

File: css/app.scss [hunk 2]
```diff
@@ -589,4 +589,15 @@ ul {
...
+  .primary-banner {
+    padding: 20px 0;
+    height: 250px;
+
+    &.main-page {
+      h2 {
+        font-size: 46px;
+      }
+    }
+  }
```
Comment (marcoow) [inline css/app.scss]: Isn't only changing the height sufficient? Also I think 200px should be enough or do we need the additional space at the bottom?

Comment (trabaldini) [reply css/app.scss]: For the work section we need at least 240px for example on an iPhone 5. 
The size fits now for an iPhone, what kind of other changes are there that i could use?

Comment (marcoow) [reply css/app.scss]: I think we can simply reduce the font size for `.primary-banner h2` to 36px on small screens.

**Review by marcoow — APPROVED** [review]

**Pull request merged**